src/examples/notifications/notifications/NotificationType.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Notification, Title, Close } from '@zendeskgarden/react-notifications';
+
+const StyledSpacer = styled.div`
+  height: ${p => p.theme.space.md};
+`;
+
+const Example = () => (
+  <>
+    <Notification type="info">
+      <Title>Info</Title>
+      Watering plants in a garden will cause them to grow.
+      <Close aria-label="Close Alert" />
+    </Notification>
+    <StyledSpacer />
+    <Notification type="warning">
```

Comment:
_Warning:_ 
**Soggy cactus**
Let the soil dry out before watering again

_Error_:
**Plums couldn't be harvested**
Make room in the basket and try again

_Success_
**Tomatoes fertilized**
Check pH before fertilizing again

_Info_:
**Partly cloudy today**
Sun-loving plants might need some extra TLC

Reply:
I opted for the veggie ipsum and tracked these in the Google doc for when we're ready to do a cohesive content update for Wells, Notification, and Alert.